### DISCUSSION TITLE
Disable TLS v1 and V2 on airflow

### DISCUSSION THREAD

**Discussion by raphaelbenja** [apache/airflow#34829] [Q&A] [answered]
Hi,

Hope you are doing well.
I have a problem on disabling TLS v1 and v2 on airflow. So is there anyome who can help me to disable it for security reason please ?

Thanks in advance.
Br

**Comment by Taragolis** [comment, marked as answer]
There is no such of [options in Airflow Webserver](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#webserver). But you could try to put Airflow Webserver behind reverse proxy, and configure allowed TLS there

- Regular http servers could do that [nginx](https://www.nginx.com/), [Apache HTTP Server,](https://httpd.apache.org/), etc
- Managed application load balancer e.g. AWS ALB or other cloud providers balancers

There is instruction how to configure nginx as reverse proxy: [Running Airflow behind a reverse proxy](https://airflow.apache.org/docs/apache-airflow/stable/howto/run-behind-proxy.html) and in additional you could easily setup allowed [ssl_protocols](http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_protocols) in nginx

**Reply by raphaelbenja** [reply]
Thanks for your help.
So, if there is no alternative I will use httpd proxy pass for this.

Br,
